src/llm_prompts.py

```diff
@@ -48,7 +48,12 @@
 
 LLM_PROMPT_SUMMARY_COMBINE_PROMPT = """
 Write a concise summary of the following text delimited by triple backquotes.
-Return your response in numbered list which covers the key points of the text and ensure that a 5 year old would understand.
```

Comment:
With these prompt summary tells you almost nothing about an article. So, I found a few good examples and picked one

Reply:
:+1: This sounds cool. To learn more, do you mind sharing some examples for the comparison? Trying to see the differences and how good those are. 

And `LLM_PROMPT_SUMMARY_COMBINE_PROMPT` and `LLM_PROMPT_SUMMARY_COMBINE_PROMPT2` are deprecated, the summary agent is now using `LLM_PROMPT_SUMMARY_COMBINE_PROMPT3`, you may want to explore/compare with that one as well.